src/sbt-test/sbt-scalafix/testkit/tests/src/test/scala/fix/InputOutputSuite.scala

```diff
@@ -0,0 +1,34 @@
+package fix
+
+import scalafix.testkit._
+import scala.util.control.NonFatal
+import org.scalatest.exceptions.TestFailedException
+
+object IntputOutputSuite {
+  def main(args: Array[String]): Unit = {
+    if (Array("--save-expect").sameElements(args)) {
```

Comment:
This looks generically useful, would it make sense to expose this somehow as an sbt-scalafix task or command where ScalafixTeskitPlugin is enabled?

Reply:
agreed! https://github.com/scalacenter/scalafix/issues/1147

Reply:
Actually, it just occured to me that I don't need to inline all this in the scripted, as I should be able to reuse `RuleSuite#main()`.

Reply:
... and no! now I remember why I didn't do it - https://github.com/scalacenter/scalafix/blob/17a187ccc3e8d8a6acb7b57b4e8fb6e254e4ecd5/scalafix-tests/unit/src/test/scala/scalafix/tests/rule/RuleSuite.scala#L8-L29 is not part of `testkit`, so it's not available here!